src/status_im/ui/screens/desktop/main/tabs/home/views.cljs

```diff
@@ -65,17 +65,54 @@
   [react/touchable-highlight {:on-press #(re-frame/dispatch [:chat.ui/navigate-to-chat chat-id])}
    [chat-list-item-inner-view (assoc chat :chat-id chat-id)]])
 
+(defn tag-view [tag {:keys [on-press]}]
+  [react/touchable-highlight {:style {:border-radius 5
+                                      :margin 2
+                                      :padding 4
+                                      :height 23
+                                      :background-color (if (= tag "clear filter")
+                                                          colors/red
+                                                          colors/blue)
+                                      :justify-content :center
+                                      :align-items :center
+                                      :align-content :center}
+                              :on-press on-press}
+   [react/text {:style {:font-size 9
+                        :color colors/white}
+                :font :medium} tag]])
+
+(defn search-input [search-filter search-filter-ref]
+  [react/view {:style {:flex 1
+                       :flex-direction :row}}
+   [react/text-input {:placeholder "Type here to search chats..."
+                      :auto-focus             true
+                      :blur-on-submit         true
+                      :style {:flex 1
+                              :color :black
+                              :height            30
+                              :margin-left       20
+                              :margin-right      22}
+                      :ref #(reset! search-filter-ref %)
+                      :default-value search-filter
+                      :on-change (fn [e]
+                                   (let [native-event (.-nativeEvent e)
+                                         text         (.-text native-event)]
+                                     (re-frame/dispatch [:search/filter-changed text])))}]])
+
 (views/defview chat-list-view []
-  (views/letsubs [home-items [:home-items]]
+  (views/letsubs [search-filter       [:search/filter]
+                  search-filter-ref   (atom nil)
```

Comment:
Why do we keep reference to the search filter react component ? I see it's passed to the `search-input` but it's not used there at all, if that's the case and I didn't miss something, please remove it.